appveyor.yml

```diff
@@ -1,22 +1,47 @@
-version: 2.0.0.{build}
+version: build.{build}
```

Comment:
Exclude these changes (for now).

Reply:
That wasn't intended... I think I had some sort of merge problem.

Reply:
Ok, will change the symbol from ppj to projectFile